job_templates/ci_job.xml.em

```diff
@@ -229,70 +241,88 @@ setlocal enableDelayedExpansion
 rmdir /S /Q ws workspace "work space"
 
 echo "# BEGIN SECTION: Determine arguments"
-set "PATH=%PATH:"=%"
-set "CI_ARGS=--force-ansi-color --workspace-path %WORKSPACE%"
-if "%CI_BRANCH_TO_TEST%" NEQ "" (
-  set "CI_ARGS=%CI_ARGS% --test-branch %CI_BRANCH_TO_TEST%"
+set "PATH=!PATH:"=!"
+set "CI_ARGS=--force-ansi-color --workspace-path !WORKSPACE!"
+if "!CI_BRANCH_TO_TEST!" NEQ "" (
+  set "CI_ARGS=!CI_ARGS! --test-branch !CI_BRANCH_TO_TEST!"
```

Comment:
> I just switched them all to !...!.

I was a bit concerned about this, because you're change their behavior just because. I was always trying to use `!...!` sparingly, defaulting to `%...%`, because that is what you want after all.

It might not break anything, and it might even been necessary if there is a new mechanism for expanding that requires them all to be, but just switching them for consistency is not right imo.